Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -135,6 +156,13 @@ const StatusBar = React.createClass({
 
     // Provide an imperative API as static functions of the component.
     // See the corresponding prop for more detail.
+
+    /**
+     * Show or hide the status bar
+     * @param {boolean} hidden The dialog's title.
+     * @param {StatusBarAnimation=} animation Optional animation when
```

Comment:
It would be nice to get types from flow instead of having to add jsdoc types too. I guess the problem is that we are not parsing the ast anymore for API docs and only using jsdoc parser.